pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import functools
+import subprocess
+from subprocess import STDOUT
+
+import pwndbg.commands
+import pwndbg.which
+
+
+
+class OnlyWithCommand(object):
+    def __init__(self, command):
+        self.cmd_name = command
+        self.cmd_path = pwndbg.which.which(command)
+
+    def __call__(self, function):
+        function.cmd_path = self.cmd_path
+
+        @pwndbg.memoize.reset_on_objfile
```

Comment:
Are you sure this works as expected? I haven't tested it but in my opinion `memoize` will make command with given arguments "return the same result", so it won't e.g. print stuff.

Imho we should memoize `call_cmd` instead.